scripts/variantstore/variant_annotations_table/GvsValidateVAT.wdl

```diff
@@ -801,6 +822,71 @@ task SubpopulationAlleleNumber {
     }
 }
 
+task DuplicateAnnotations {
+    input {
+        String query_project_id
+        String fq_vat_table
+        String last_modified_timestamp
+    }
+    # there are vids with duplicate annotations
+    # make sure there aren't matching sites with mis-matched ANs and matching variants with mis-matched ACs
+
+    String pf_file = "pf.txt"
+    String results_file = "results.txt"
+
+    command <<<
+        # Prepend date, time and pwd to xtrace log entries.
+        PS4='\D{+%F %T} \w $ '
+        set -o errexit -o nounset -o pipefail -o xtrace
+
+        echo "project_id = ~{query_project_id}" > ~/.bigqueryrc
+
+        bq query --nouse_legacy_sql --project_id=~{query_project_id} --format=csv 'SELECT * from
+        (SELECT contig, position, gvs_all_an, COUNT(DISTINCT gvs_all_an) AS an_count FROM `~{fq_vat_table}`
+        group by contig, position, gvs_all_an)
+        where  an_count >1' > bq_an_output.csv
```

Comment:
It should be possible to simplify this and the query below with a `HAVING` (i.e. without a nested query) as was [done previously](https://github.com/broadinstitute/gatk/pull/8175/files#diff-df904386924b83f1524c95c807a2ceb451e53bd91af830712669bc613ad1217fL389). However I'm not actually sure what the `COUNT(DISTINCT gvs_all_an)` is trying to do...

```suggestion
        bq query --nouse_legacy_sql --project_id=~{query_project_id} --format=csv '
          SELECT contig, position, gvs_all_an, COUNT(DISTINCT gvs_all_an) AS an_count
          FROM `~{fq_vat_table}`
          GROUP BY contig, position, gvs_all_an
          HAVING an_count > 1
          ' > bq_an_output.csv
```

Reply:
I'm trying to make sure that for multiple rows with the same VID, there aren't multiple values of AN (and below the AC!)